frameworks/CSharp/aspnetcore/aspcore-ado-my.dockerfile

```diff
@@ -1,11 +1,10 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.0.100-preview7 AS build
```

Comment:
Better to just use 
```
FROM mcr.microsoft.com/dotnet/core/sdk:3.0 AS build 
```
Then it will automatically update without needing to make more PRs?

Reply:
I intentionally decided not to. I don't want the benchmarks to be unstable, either because there is a breaking change, or because the perf is volatile.

Reply:
In the end we decide to use 3.0. Hope you like it